Code samples for API usage/sampleCode.py

```diff
@@ -0,0 +1,164 @@
+            "/Videos?name={}&privacy=Private&videoUrl={}&indexingPreset=AdvancedVideo&accessToken={"
+            "}&sendSuccessEmail=True&streamingPreset=NoStreaming"
+        ).format(cls.location, cls.account_id, video_id, video_url, access_token)
+
+        # Make request and catch errors
+        response = requests.post(url=video_indexer_url, headers=headers)
+        if response.status_code == 200:
+            video_indexer_id = response.json()["id"]
+            return video_indexer_id
+        # If the access token has expired get a new one
+        if response.status_code == 401:
+            print("[*] Access token has expired, retrying with new token.")
+            access_token = cls.get_access_token()
+            video_indexer_new_url = """https://api.videoindexer.ai/{}/Accounts/{}/Videos?name={}&privacy=Private&videoUrl={}&accessToken={}&sendSuccessEmail=True&streamingPreset=NoStreaming""".format(
+                cls.location,
+                cls.account_id,
+                video_id,
+                video_url,
+                access_token,
+            )
+            response = requests.post(url=video_indexer_new_url, headers=headers)
+            if response.status_code == 200:
+                video_indexer_id = response.json()["id"]
+                return video_indexer_id
+            else:
+                print("[*] Error after retrying.")
+                print(
+                    "[*] Response : {} {}".format(response.status_code, response.reason)
+                )
+        # If you are sending too many requests
+        if response.status_code == 429:
+            print("[*] Throttled for sending too many requests.")
+            time_to_wait = response.headers["Retry-After"]
+            print("[*] Retrying after {} seconds".format(time_to_wait))
+            sleep(int(time_to_wait))
+            response = requests.post(url=video_indexer_url, headers=headers)
+            if response.status_code == 200:
+                video_indexer_json_output = response.json()
+                return video_indexer_json_output
+            else:
+                print("[*] Error after retrying following throttling.")
+                print(
+                    "[*] Response : {} {}".format(response.status_code, response.reason)
+                )
+        else:
+            print("[*] Error when calling video indexer API.")
+            print("[*] Response : {} {}".format(response.status_code, response.reason))
+
+    @classmethod
+    def get_indexed_video_data(cls, video_id, access_token):
+        """
+        Retrieves data on the video after analysis from the Video Indexer API.
+        :param video_id: string, unique identifier for the indexed video.
+        :param access_token: string, required to use the API.
+        :return video_indexer_json_output: JSON, analysed video data.
+        """
+
+        # Set request headers and url
+        headers = {
+            "Ocp-Apim-Subscription-Key": cls.subscription_key,
```

Comment:
Not required

Reply:
@ika-microsoft  : is this PR ready for merge ?